docs/collections/documents/tutorials/c8ql-docs-queries.md

```diff
@@ -0,0 +1,294 @@
+---
+sidebar_position: 80
+title: C8QL Docs Queries Example
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import Prerequisites from '../../../_partials/_prerequisites-sdk-api-key.md';
+import GetStarted from '../../../_partials/_get-started-steps.md';
+
+This page demonstrates how you can use C8QL and the Macrometa SDK to run CRUD operations on document store collection records. For more information about C8QL queries, refer to [C8QL](../../../queries/c8ql/) and [Queries](../../../queries/).
```

Comment:
@diana-macrometa I have updated code samples to have the same output format but we need to change the above description where it says Macrometa SDK.. we are not using SDK in this example we are using APIs with Python and Javascript

Reply:
Updated. Anything else?

Reply:
Good catch! I changed them to the correct partials.